src/mlpack/tests/feedforward_network_2_test.cpp

```diff
@@ -65,7 +65,8 @@ TEST_CASE("RBFNetworkTest", "[FeedForwardNetworkTest]")
 {
   // Load the dataset.
   arma::mat trainData;
-  data::Load("thyroid_train.csv", trainData, true);
+  if (!data::Load("thyroid_train.csv", trainData))
+    Fail("Cannot open thyroid_train.csv");
```

Comment:
```suggestion
    FAIL("Cannot open thyroid_train.csv");
```

Reply:
I thought it was strange that the tests were still passing, but then I noticed this file wasn't being compiled.  I opened #2851, which we should merge and then make sure this still works before merging this one. :+1: